cloudwatch-logs-to-logstash/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $TEST_FUNC_NAME ]; then
+    echo 'TEST_FUNC_NAME is unset';
+    exit 1
+fi
+
+if [ -z $PROD_FUNC_NAME ]; then
+    echo 'PROD_FUNC_NAME is unset';
+    exit 1
+fi
+
+cd lambda
+npm install
+
+npm test
+
+# TODO get or create config.json
```

Comment:
Difficult one. I wonder whether we really do need to worry about secrets in the CI environment?